lib/constructs/interface.js

```diff
@@ -1420,7 +1463,7 @@ class Interface {
     const { idl, name } = this;
 
     this.str += `
-      exports.install = function install(globalObject) {
+      exports.install = (globalObject, globalName) => {
```

Comment:
After re‑reading the spec for `[Global]` and `[Exposed]`, I realised that this needs `globalNames` instead of `globalName`.